test/fail_compilation/test22899.c

```diff
@@ -16,4 +15,4 @@ void fn()
 {
     int x;
     x = sizeof( (mytype_t) );
-}
+}}
```

Comment:
this added brace adds a (second) syntax error on top of the one being tested

it should probably be moved to an `.i` file instead if one of the preprocessors complained about it

Reply:
I prefer it to work with the preprocessor.